test-cases/longevity/longevity-large-collections-1h.yaml

```diff
@@ -0,0 +1,25 @@
+test_duration: 60
+stress_cmd: [
+  "JVM_OPTION='-Xms8033M -Xmx8033M -Xmn400M' cassandra-stress user profile=/tmp/large_collections.yaml ops'(insert=3,read1=1,update1=1)' cl=QUORUM duration=20m -mode cql3 native -rate threads=8"
+]
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3en.3xlarge'
+
+nemesis_class_name: 'NoOpMonkey'
+nemesis_seed: '012'
+nemesis_interval: 15
+
+user_prefix: 'longevity-large-collections-1h'
+space_node_threshold: 64424
+
+# Disable hinted handoff to avoid a known issue:
+# https://github.com/scylladb/scylla/issues/4935
+hinted_handoff: 'disabled'
+
+append_scylla_yaml: |
+  compaction_collection_items_count_warning_threshold: 9000
```

Comment:
please add a comment explain what it does and why we need it.

Reply:
Removed this file.